pkg/kubelet/cm/devicemanager/topology.go

```diff
@@ -0,0 +1,124 @@
+package devicemanager
```

Comment:
so in the cpumanager you have the toplogy_hints.go, so maybe we should rename this file to topology_hints.go as well (for consistency)

Reply:
+1 good suggestion will update